native/package.json

```diff
@@ -122,9 +122,11 @@
     "jest-transform-stub": "^2.0.0",
     "jetifier": "^1.6.5",
     "metro-react-native-babel-preset": "^0.59.0",
+    "react-native-flipper": "^0.64.0",
```

Comment:
Flipper is already integrated in our current react-native setup: https://fbflipper.com/docs/getting-started/react-native

Reply:
Does not hurt though, does it?

Reply:
This is not how flipper should be integreated after 0.63 afaik. This could lead to duplicate binaries or whatever

Reply:
redux-flipper does not work without an explicit dependency.

react-native-flipper is also included in the official example here: https://github.com/facebook/flipper/blob/master/react-native/ReactNativeFlipperExample/package.json
and mentioned here as well: https://github.com/facebook/flipper/tree/master/react-native/react-native-flipper#installation

Reply:
Alright I checked what is happening here. react-native-flipper just exposes bindings for flipper. That means flipper works without that package well, but redux-flipper requires the bindings.